### ISSUE TITLE
inferelator should run from any directory

### ISSUE THREAD

**Issue by AaronWatters** [simonsfoundation/inferelator_ng#53]
I think right now the code tends to assume that the inferelator runs from the repository root directory.  Inferelator should be runnable from any directory for more clean data organization.
In particular I just merged  #52 which loads a github repo but doesn't install the package globally.

**Comment by NicoDeVeaux** [comment]
Agree. I think we should circle back up on this when we start talking about the requirements for releasing and packaging the software
